Jacobi/Sensei/C/solution/JacobiDataAdaptor.cxx

```diff
@@ -252,17 +360,60 @@ vtkDataObject* JacobiDataAdaptor::GetMesh(bool vtkNotUsed(structure_only))
     int n_ranks = 1;
     MPI_Comm_size(MPI_COMM_WORLD, &n_ranks);
 
+#ifdef USE_IMAGE_DATA
+    int Extent[6];
+    double Origin[3], Spacing[3];
+    Extent[0] = this->sim->rankx*this->sim->bx;
+    Extent[1] = Extent[0] + this->sim->bx + 2 - 1;
+    Extent[2] = this->sim->ranky*this->sim->by;
+    Extent[3] = Extent[2] + this->sim->by + 2 - 1;
+    Extent[4] = 0;
+    Extent[5] = 0;
+
+    Spacing[0] = this->sim->cx[1] - this->sim->cx[0];
+    Spacing[1] = this->sim->cy[1] - this->sim->cy[0];
+    Spacing[2] = 0.;
+
+    // Origin on rank 0 using cx[0], cy[0] is zero.
```

Comment:
@burlen @BradWhitlock

Sorry guys, I have been in backlog mode ever since I finished the tutorial with Burlen, teaching a 3rd day event, then attending a class myself. I cannot find the time to properly interact with you until after April 9 (will be on vacation for the Easter week). Cheers. Jean